Mathlib/AlgebraicGeometry/EllipticCurve/Point.lean

```diff
@@ -75,6 +75,9 @@ local macro "map_simp" : tactic =>
 
 universe u v w
 
+-- Disabled locally for performance reasons
+attribute [-instance] StrictOrderedSemiring.to_charZero
```

Comment:
In addition to the disappointing [benchmark results](https://github.com/leanprover-community/mathlib4/pull/6548#issuecomment-1681196667) I think this is a pretty big red flag.